### PR TITLE
Updated README usage information

### PR DESCRIPTION
**Pull request by telday**
### What does this PR do?
Updated ReadMe to include more detailed information about client setup/configuration.

#### Change log
- [ ] The CHANGELOG has been updated, or
- [x] This PR does not include user-facing changes and doesn't require a CHANGELOG update

#### Test coverage
- [ ] This PR includes new unit and integration tests to go with the code changes, or
- [x] The changes in this PR do not require tests

#### Documentation
- [x] Docs (e.g. `README`s) were updated in this PR, and/or there is a follow-on issue to update docs, or
- [ ] This PR does not require updating any documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +24,374 @@ its own version allows changes to be made to the client outside of updates to th
...
+2. Go into the cloned repository with `cd conjur-sdk-java`
+3. Run `mvn package -DskipTests` to generate a JAR file. The output `.jar` files will be located
+   in the `target` directory of the repo
+
+_NOTE:_ The above command runs `mvn package` without running the integration tests, since
+these require access to a Conjur instance. You can run the integration tests with
+the `./bin/test_integration` script. For more information on how to run the tests, take a look at
+our [Contributing](https://github.com/cyberark/conjur-api-java/blob/main/CONTRIBUTING.md) guide.
+
+4a. For Intellij, Follow the steps outlined [here](https://www.jetbrains.com/help/idea/library.html)
```
Comment (john-odonnell) [inline README.md:103]: 4a, b and c are a single, unbroken line in the rendered MD: https://github.com/cyberark/conjur-sdk-java/tree/update-usage-documentation#using-the-jarfile

Also, the code blocks below are rendered as one continuous block, including backticks.

**Review by john-odonnell — CHANGES_REQUESTED** [review]
Looks good! I've made a comment (above, accidentally outside of the review scope).

**Review by john-odonnell — APPROVED** [review]
LGTM 👍

**Comment by alexkalish** [comment]
Thanks, @telday!!  Looks great.

File: README.md [hunk 2]
```diff
@@ -24,6 +24,381 @@ its own version allows changes to be made to the client outside of updates to th
...
+
+```java
+import com.cyberark.conjur.sdk.ApiClient;
+import com.cyberark.conjur.sdk.Configuration;
+import com.cyberark.conjur.sdk.endpoint.AuthenticationApi;
+
+// Get the systems default API client
+ApiClient client = Configuration.getDefaultApiClient();
+// Given you already have an api key you can just set the clients value
+client.setUsername("username");
```
Comment (sigalsax) [inline README.md:356]: came across this PR and just popping in here for a second :)
Is this sustainable if we keep adding new ways to login to the client? 

What about items like account, cert_file, and url. Can they be set in this way as well?

**Pull request merged**